admin/index.html

```diff
@@ -0,0 +1,12 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Content Manager</title>
+</head>
+<body>
+  <!-- Include the script that builds the page and powers Netlify CMS -->
+  <script src="https://unpkg.com/netlify-cms@^2.0.0/dist/netlify-cms.js"></script>
```

Comment:
@dylanirlbeck It's bad security practice to remotely load .js . Let's change this to self-hosting the netlifycms js as an asset in the repo so we can manage dependency upgrades in the future